directanswercards/card_component.js

```diff
@@ -40,64 +40,40 @@ BaseDirectAnswerCard["{{componentName}}"] = class extends ANSWERS.Component {
   }
 
   onMount() {
-    let feedbackFormSelector = '.js-HitchhikerDirectAnswerCard-feedbackForm';
+    this.addFeedbackListeners();
+
+    const rtfElement = this._container.querySelector('.js-yxt-rtfValue');
+    rtfElement && rtfElement.addEventListener('click', e => this._handleRtfClickAnalytics(e));
+  }
+
+  addFeedbackListeners() {
+    const feedbackFormSelector = '.js-HitchhikerCard-feedbackForm';
     let feedbackFormEl = this._container.querySelector(feedbackFormSelector);
     if (feedbackFormEl) {
       // For WCAG compliance, the feedback should be a submittable form
       feedbackFormEl.addEventListener('submit', (e) => {
-        let formTargetEl = e.target;
-        let checkedValue = formTargetEl.querySelector('input:checked').value === 'true';
+        const formTargetEl = e.target;
+        const isGood = formTargetEl.querySelector('input:checked').value === 'true';
 
-        this.reportQuality(checkedValue);
+        this.reportQuality(isGood);
         this.updateState({
-          'feedbackSubmitted': true
+          feedbackSubmitted: true
         });
       });
 
-      let thumbSelectorEls = this._container.querySelectorAll('.js-HitchhikerDirectAnswerCard-thumbInput');
+      let thumbSelectorEls = this._container.querySelectorAll('.js-HitchhikerCard-thumbInput');
```

Comment:
Do we need to worry about backwards compatibility, if somebody forked one fo the direct answer cards?

Reply:
(If we already asked Rose and got her approval then we good here!)

Reply:
This change is because the direct answer cards are now using the same shared feedback partial as results cards. So, this would be a problem for someone forking a previous version of the direct answer card

Reply:
I think Tom had mentioned that he would let Rose know that we were switching the direct answer cards to use the shared partial

Reply:
It's not an issue for there to be a shared feedback partial between DA and Results cards. But, we can't break any custom DA cards a user has created. If someone made a custom DA card before this change, that means they are using now invalid classnames. So, feedback submission wouldn't work. 

We can still have a shared partial and be backwards compatible. Instead of using the class name `js-HitchhikerCard` for both, we would make it dynamic. When a DA card invokes the partial, the class names would have a `.js-HitchhikerDirectAnswerCard` prefix.

Reply:
Ok, I'll fix that